src/UserSwitch/UserSwitch.tsx

```diff
@@ -194,6 +194,7 @@ export const UserSwitch = ({
         aria-controls={menuId}
         aria-expanded={Boolean(anchorEl)}
         tabIndex={0}
+        data-umami-event='header-avatar'
```

Comment:
Should this be hardcoded? the userswitch could be used somewhere else than the header.